yarn-install/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
     - name: find location of global yarn cache
       id: yarn-cache
       if: steps.cache-modules.outputs.cache-hit != 'true'
-      run: echo ::set-output name=dir::$([[ "$(yarn --version)" =~ "1.*" ]] && yarn cache dir || yarn config get cacheFolder)
+      run: echo "dir=$([[ "$(yarn --version)" =~ "1.*" ]] && yarn cache dir || yarn config get cacheFolder)" >> $GITHUB_OUTPUT
```

Comment:
now i'm curious if this will follow the same escaping rules as before, re. the double quotes in and out of the string arg and the `$` invocation :)

Reply:
I ran it locally and it seems to work as expected :)